deploy/examples/cluster-test.yaml

```diff
@@ -57,6 +43,13 @@ spec:
     mgr: system-cluster-critical
   disruptionManagement:
     managePodBudgets: true
+  cephConfig:
```

Comment:
There must be some yaml formatting issue. The CI tests that use cluster-test.yaml are failing with this error:
```
Error from server (BadRequest): error when creating "cluster-test.yaml": CephBlockPool in version "v1" cannot be handled as a CephBlockPool: strict decoding error: unknown field "spec.monitoring"
```

Reply:
I can't repro this either, so it's not clear what is causing this. I don't see what is wrong with the cluster-test.yaml.  Perhaps the type `map[string]map[string]string` is just too complex for the deserializer and it sometimes gets confused. We might explore just making this a `string`, and use the multi-line string to pass the raw text to the assimilate-conf. This would mean using the format:
```
cephConfig: |
  [global]
  setting = val
```
instead of well-structured yaml
```yaml
cephConfig:
  global:
    setting: val
```
But we expect this is an advanced feature anyway, and it will also simplify the transition for users converting from the existing ceph.conf

Reply:
I just dug further and the `yq` commands in the github actions helper are adding the `monitoring.enabled: true` to the CephblockPool because of the wrong document index.

I have pushed a fix for all occurrences in that script to correctly set the cluster options on the cluster "document".

Reply:
Ah much better fix, thanks!